docs/ops/movement/SpaceToDepth_1.md

```diff
@@ -49,15 +28,32 @@ If `mode = depth_first`:
 
 **Inputs**
 
-*   **1**: `data` - input tensor of type *T* with rank >= 3. **Required.**
+*   **1**: `data` - input tensor of any type with rank >= 3. **Required.**
 
 **Outputs**
 
-*   **1**: permuted tensor of type *T* and shape `[N, C * (block_size ^ K), D1 / block_size, D2 / block_size, ..., DK / block_size]`.
+*   **1**: permuted tensor with shape `[N, C * (block_size ^ K), D1 / block_size, D2 / block_size, ..., DK / block_size]`.
+
+**Detailed description**
+
+*SpaceToDepth* operation permutes element from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input spatial dimensions `D1, D2, ..., DK` are moved to the new depth dimension. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#SpaceToDepth) for an example of the 4D input tensor case.
+
+The operation is equivalent to the following transformation of the input tensor `data` with `K` spatial dimensions of shape `[N, C, D1, D2, ..., DK]` to *Y* output tensor. If `mode = blocks_first`:
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ... , DK/block_size, block_size])
+
+    x'' = transpose(x',  [0,  3, 5, ..., K + (K + 1), 1,  2, 4, ..., K + K])
 
-**Types**
```

Comment:
This file contains some probably merge artifacts like removing **Types**, while this section has been added during SpecToDepth spec update.